tests/system_tests/appstart_systemtests.py

```diff
@@ -80,15 +88,23 @@ def _endpoint_test(self):
         self.assertEqual(res.status_code,
                          200,
                          '%s failed with error \"%s\"' %
-                         (handler.__name__, res.text))
+                         (endpoint, res.text))
     _endpoint_test.__name__ = 'test_%s_endpoint' % endpoint.strip('/')
     return _endpoint_test
 
 
 if __name__ == '__main__':
     logging.getLogger('appstart').setLevel(logging.INFO)
+
+    # Sync with urls in services_test_app.py
+    # Keeping handler as None for later on customizing of tests
+    urls = [('/datastore', None),
+        ('/logging', None),
+        ('/memcache', None)]
+
```

Comment:
Got it :-)  Can you also remove the "None"s here and also the handler parameter to make_endpoint_test()?